src/site/content/en/blog/webcomponents-template/index.md

```diff
@@ -322,9 +323,7 @@ full featured is always a good thing in my book.
 ## Additional resources
 
 - [WhatWG Specification][spec-link]
-- [Introduction to Web Components](http://w3c.github.io/webcomponents/explainer/#template-section)
+- [Introduction to Web Components](https://www.w3.org/TR/2012/WD-components-intro-20120522/#template-section)
```

Comment:
The link intentionally points to an older version of explainer that contains overview of templates.